_sass/vrsite/gallery.scss

```diff
@@ -43,22 +43,25 @@
 		.title {
 			text-align: center;
 		}
+
 		.desc {
 			overflow: hidden;
 			text-overflow: ellipsis;
```

Comment:
Should we hide the overflow at all? Given that users will have no way of seeing the expanded text

Reply:
I changed overflow to scroll and so now the description for each person doesn't have character limits and the text is scrollable to read.